operator/docs/user-guides/recover_ingester_pods_in_zone_failure_outages.md

```diff
@@ -0,0 +1,97 @@
+
+This document outlines steps that can be taken to recover ingester pods when there is a zone failure. Ingester pods are deployed as a part of a Statefulset. The Ingester Statefulset also has PVCs associated with the pods which are dynamically provisioned through the use of StorageClass. Each Ingester Pod and its associated PVCs are deployed in the same zone.
+
+When a zone failure occurs in a cluster, the StatefulSet controller will automatically attempt to recover the affected pods in the failed zone. This document outlines the additional manual intervention required to make sure that the ingester pods are succesfully recreated in a new zone.
+
+* Detect Zone Failure - The control plane and cloud provider integration should mark nodes in the failed zone.
+
+* Reschedule Pods: The Ingester StatefulSet controller will automatically attempt to reschedule the pods that were running in the failed zone to nodes in another zone.
+  
+* Since the Statefulset has PVCs which are also in the failed zone, automatic reschedule of the Ingester pods to a different zone will not work. See - [Storage access for zones](https://kubernetes.io/docs/setup/best-practices/multiple-zones/#storage-access-for-zones). Manual intervention is required at this point to delete the old pvcs in the failed zone to allow succesful recreation of the Ingester Pod & Pvc in the new zone.
+
+    Example showing an ingester pod that is in a pending state, after the statefulset has unsuccesfully tried to reschedule it to a different zone.
+    Ingester-1 is in Pending state because it cannot be scheduled in a new node in a different zone:
+
+    ```console
+    oc get pods -n openshift-logging
+    NAME                                           READY   STATUS    RESTARTS   AGE
+    lokistack-dev-ingester-0                       1/1     Running   0          4m21s
+    lokistack-dev-ingester-1                       0/1     Pending   0          44s
+    ```
+
+    This is because the pvc for ingester-1 is in the old zone
+
+    ```console
+    oc get pvc -n openshift-logging
+    NAME                                    STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+    storage-lokistack-dev-ingester-0        Bound     pvc-1be8a2c2-f594-4ce7-9139-d57de6e06ce9   10Gi       RWO            gp3-csi        12m
+    storage-lokistack-dev-ingester-1        Pending                                                                        gp3-csi        8m24s
+    wal-lokistack-dev-ingester-0            Bound     pvc-0239dca6-cf2c-4b67-94a8-e2e0fb94509d   150Gi      RWO            gp3-csi        12m
+    wal-lokistack-dev-ingester-1            Pending                                                                        gp3-csi        8m24s
+    ```
+
+    Delete the PVCs, followed by the pod. The pod and new pvcs should now be recreated in an available zone because the statefulset has a set number of replicas
```

Comment:
```suggestion
    Delete the PVCs, followed by the pod. The pod and new PVCs should now be recreated in an available zone because the StatefulSet has a set number of replicas
```